test/algodclientv2_test.go

```diff
@@ -362,3 +363,8 @@ func parsedDryrunResponseShouldHave(key string, action int) error {
 	}
 	return nil
 }
+
+func mockHttpResponsesInLoadedFrom(jsonfiles, directory string) error {
+	fullPath := path.Join("./features/unit/", directory)
```

Comment:
are these also in `./features/resources`?

Reply:
It is working in Travis. And I do not decide where travis gets them from. So I must be doing something right.

Reply:
It's one of the "backwards compatibility" considerations. Once everything has moved to `resources` (hopefully after responses v2 are implemented for all SDKs) we'll delete the duplicate response files in `unit`.